src/Controllers/MoviesController.cs

```diff
@@ -123,10 +123,11 @@ public IActionResult Edit(MovieViewModel mvm)
         [HttpGet("Details")]
         public IActionResult Details(int id)
         {
-            var movie = UnitOfWork.Movies.Get(id);
+            var movie = UnitOfWork.Movies.GetMovieWithComments(id);
+            var comments = movie.Comments.Select(c => new CommentViewModel{ Id = c.Id, Text = c.Text } ).ToList();
             if (movie == null)
                 return NotFound();
-            return View(new MovieViewModel { Title = movie.Title, ReleaseDate = movie.ReleaseDate, Genre = movie.Genre, Price = movie.Price, Rating = movie.Rating } );
+            return View(new MovieViewModel { Id = movie.Id, Title = movie.Title, ReleaseDate = movie.ReleaseDate, Genre = movie.Genre, Price = movie.Price, Rating = movie.Rating, Comments = comments } );
```

Comment:
its ok, but you can avoid to declare `comments` variable.
`return View(new MovieViewModel`
`{`
`    Id = movie.Id,`
`   Title = movie.Title,`
`    ReleaseDate = movie.ReleaseDate,` 
`   Genre = movie.Genre,` 
`    Price = movie.Price,` 
`    Rating = movie.Rating,` 
`    Comments = movie.Comments.Select(c => new CommentViewModel{ Id = c.Id, Text = c.Text } ),`
`});`